shared/src/main/kotlin/org/javacs/kt/SourceExclusions.kt

```diff
@@ -16,13 +14,15 @@ class SourceExclusions(
     val excludedPatterns = (listOf(
         ".git", ".hg", ".svn",                                                      // Version control systems
         ".idea", ".idea_modules", ".vs", ".vscode", ".code-workspace", ".settings", // IDEs
-        "bazel-*", "bin", "build", "node_modules", "target",                        // Build systems
+        "bazel-*", "bin", "build", "node_modules",
+        "target/*", "!target/generated-sources", "!target/generated-sources/**"     // Build systems
```

Comment:
Should it not be better to just fix the TODO above

Reply:
Not sure if it makes sense to get these form .gitignores etc, this assumes that such files exist.
In this case we need to have some sensible defaults.